pages/index.vue

```diff
@@ -13,12 +13,14 @@
 
 <script lang="ts">
 import Vue from "vue";
+import i18n from 'vue-i18n';
```

Comment:
This import should not be necessary AFAIK

Reply:
Yeah, I added it while trying something and forgot to remove it.